tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -1067,27 +1067,42 @@ TEST (nnstreamer_capi_singleshot, invoke_01)
   }
 
   /* generate dummy data */
-  input = ml_model_allocate_tensors_data (&in_info);
+  input = ml_util_allocate_tensors_data (&in_info);
   EXPECT_TRUE (input != NULL);
 
+  status = ml_util_get_last_error ();
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
   output1 = ml_model_inference (model, input, NULL);
   EXPECT_TRUE (output1 != NULL);
-  ml_model_free_tensors_data (output1);
 
-  output2 = ml_model_allocate_tensors_data (&out_info);
+  status = ml_util_get_last_error ();
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  ml_util_free_tensors_data (&output1);
```

Comment:
`output1` is freed by `ml_util_free_tensors_data()`, then `output1` is compared with `output1` at 1092 line.

Is it correct?

Reply:
Sorry. I misunderstood. Good job!